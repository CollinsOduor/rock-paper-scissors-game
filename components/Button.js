import styled from "styled-components";

export const Button = styled.button`
  display: flex;
  justify-content: center;
  align-items: center;
  color: #000000;
  background-color: #ffffff;
  border: 2px solid #ffffff;
  border-radius: 5px;
  padding: 2%;
  height: fit-content;
  font-weight: bold;
  font-size: large;
`;
