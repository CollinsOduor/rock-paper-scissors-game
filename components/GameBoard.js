import styled from "styled-components";

export const GameBoard = styled.div`
  display: grid;
  width: 60%;
  margin: auto;
  height: 50vh;
  grid-gap: 10%;
  grid-template-areas:
    "paper . scissors"
    ". rock .";
  justify-content: center;
`;
