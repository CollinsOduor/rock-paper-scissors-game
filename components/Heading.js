import styled from "styled-components";

export const Heading = styled.div`
  display: flex;
  width: 100%;
  max-width: 800px;
  margin: auto;
  border: 4px solid #5d6c86;
  border-radius: 6px;
  align-items: center;
  justify-content: space-between;
  padding: 0 2%;
`;
