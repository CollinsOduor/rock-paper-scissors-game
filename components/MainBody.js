import styled from "styled-components";

export const MainBody = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  min-height: 100vh;
  padding: 1% 2% 5% 2%;
  background: radial-gradient(at top center, #1f3656, #171f40);
`;
