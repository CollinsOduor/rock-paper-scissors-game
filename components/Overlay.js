import styled from "styled-components";

export const Overlay = styled.div`
  width: fit-content;
  padding: 2%;
  height: fit-content;
  display: flex;
  flex-direction: column;
  row-gap: 3vh;
  justify-content: center;
  align-items: center;
  z-index: 3;
  background-color: #ffffff;
  position: absolute;
  top: 25vh;
  border-radius: 3px;
`;
