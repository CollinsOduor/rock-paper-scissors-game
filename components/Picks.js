import styled from "styled-components";

export const Picks = styled.div`
  display: flex;
  justify-content: space-around;
  align-items: center;
  column-gap: 5vw;
  height: 50vh;
  color: #ffffff;
  width: fit-content;
`;
