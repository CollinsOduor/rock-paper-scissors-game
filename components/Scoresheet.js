import styled from "styled-components";

export const Scoresheet = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  color: black;
  background-color: #ffffff;
  border: 2px solid #ffffff;
  border-radius: 5px;
  padding: 2%;
`;
