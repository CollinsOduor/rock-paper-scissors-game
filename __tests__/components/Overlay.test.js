import React from "react";
import renderer from "react-test-renderer";
import { Overlay } from "../../components/Overlay";

describe("Overlay", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Overlay />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
