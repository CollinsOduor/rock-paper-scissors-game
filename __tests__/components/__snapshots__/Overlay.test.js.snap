// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay renders correctly 1`] = `
<div
  className="Overlay-sc-18ugkme-0 hEIcxU"
/>
`;
